core/mocks/trezortranslate_keys.pyi

```diff
@@ -144,7 +144,7 @@ class TR:
     cardano__collateral_input_index: str = "Collateral input index:"
     cardano__collateral_output_contains_tokens: str = "The collateral return output contains tokens."
     cardano__collateral_return: str = "Collateral return"
-    cardano__confirm: str = "Confirm:"
+    cardano__confirm: str = ""
```

Comment:
why not remove these completely?

Reply:
I think there were issues last time I tried to remove strings due to indexes of the other strings not matching anymore...

Reply:
@ibz To remove a string:
- delete it from lang JSONs (`en.json`, `cs.json`, ...) 
- keep the key in `order.json`
- regenerating should keep the entry in rust enum (with empty `""`) and you'll see it removed from `mocks/trezortranslate-keys.pyi`

Can you try?

Reply:
@obrusvit You are right, I remember now!